### PR TITLE
Adding StackedCubedSphereTopology type

### PR DESCRIPTION
**Pull request by jkozdon**
This pull requests adds a `StackedCubedSphereTopology` type.

### CODE REVIEW

File: src/ClimaAtmos/Dycore/src/vtk.jl [hunk 1]
```diff
@@ -26,7 +26,7 @@ end
 #=
 This is the 2D WriteMesh routine
 =#
-function writemesh(base_name, x, y; fields=(), realelems=1:size(x)[end])
+function writemesh(base_name, x, y; z=nothing, fields=(), realelems=1:size(x)[end])
   @assert size(x) == size(y)
   (Nqr, Nqs, ~) = size(x)
```
Comment (simonbyrne) [inline src/ClimaAtmos/Dycore/src/vtk.jl]: should the `~` here be a `_`?

Comment (lcw) [reply src/ClimaAtmos/Dycore/src/vtk.jl]: I think so.  I am guessing this is a MATLABism.  I submitted a separate pull request, https://github.com/climate-machine/CLIMA/pull/103, that makes the change you suggested.

Comment (simonbyrne) [reply src/ClimaAtmos/Dycore/src/vtk.jl]: Somewhat annoyingly, it doesn't appear to be documented, but the idea is that `_` can only appear as a variable on the left-hand side of an assignment.

Comment (jkozdon) [reply src/ClimaAtmos/Dycore/src/vtk.jl]: fixed in #103

**Pull request merged**